source/array-slice.d.ts

```diff
@@ -0,0 +1,62 @@
+import type {IsNegative} from './numeric';
+import type {Add, Lt, Lte, Max} from './internal';
+
+/**
+Returns a copy of a section of an array, just like `Array.slice` in js.
```

Comment:
The description needs some work and it should be identifical to the one in the readme.

Reply:
I have no idea about the new description, please modify it directly